lib/src/http/common/request.dart

```diff
@@ -149,8 +150,9 @@ abstract class CommonRequest extends Object
   /// Whether or not to send the request with credentials.
   bool _withCredentials = false;
 
-  /// [Client] instance from which this request was created. Used in [clone] to
-  /// correctly tie the clone to the same client.
+  /// HttpClient instance from which this request was created. Used in [clone]
```

Comment:
Every instance of client in the codebase right now is typed as `Client`, but is actually an instance of `HttpClient` (this is for backwards compatibility).

You can see this here: https://github.com/Workiva/w_transport/blob/master/lib/src/http/client.dart#L32-L33 – the `Client` factory actually returns an `HttpClient` instance.